backend/src/main/java/heartbeat/controller/HealthController.java

```diff
@@ -0,0 +1,20 @@
+package heartbeat.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.actuate.health.HealthEndpoint;
+import org.springframework.boot.actuate.health.Status;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/health")
+@RequiredArgsConstructor
+public class HealthController {
+    private final HealthEndpoint healthEndpoint;
+
+    @GetMapping
+    public Status getHealthStatus() {
+        return healthEndpoint.health().getStatus();
```

Comment:
Return a string or structure directly

Reply:
Status is a object with a field code(serialized with name status), sample response for this endpoint is {"status": "UP"}